### ISSUE TITLE
Why weighting which is irrelevant to sigmas, is required for computing epsilon_loss?

### ISSUE THREAD

**Issue by Xingyuxingyu** [Microtreei/TSD-SR#8]
Hello,
First of all, thank you for your job, which works for me.
I have a question. Why is weighting introduced when calculating loss? The default weighting_scheme is "logit-normal", which is irrelevant to sigmas.

epsilon_loss = 0.5 * weighting.view(-1,1,1,1) *  F.mse_loss(model_pred_reg.float(), latent_stu.detach().float(), reduction='mean')

Also, why do you need to multiply by a factor of 0.5?

**Comment by Xingyuxingyu** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，已收到您的邮件。我会尽快给您回复。
